sys-devel/llvm/llvm9-9.0.1.recipe

```diff
@@ -474,7 +474,7 @@ BUILD_PREREQUIRES="
 	cmd:ld$secondaryArchSuffix
 	cmd:make
 	cmd:ninja
-	cmd:python2
+	cmd:python3
 	cmd:python$PYTHON3_VERSION
```

Comment:
Does it need `cmd:python3` and `cmd:python$PYTHON3_VERSION`?

Reply:
Good point, didn't caught my eye there ;)